pkg/config/nodetreemodel/config.go

```diff
@@ -453,9 +453,11 @@ func (c *ntmConfig) buildEnvVars() {
 		envkey := pair[0]
 		envval := pair[1]
 
-		if configKey, found := c.configEnvVars[envkey]; found {
-			if err := c.insertNodeFromString(root, configKey, envval); err != nil {
-				envWarnings = append(envWarnings, fmt.Sprintf("inserting env var: %s", err))
+		if configKeyList, found := c.configEnvVars[envkey]; found {
```

Comment:
Should we sort the result from `os.Environ` to have a predicted behavior in case N env vars are linked to the same key ?

Also should we raise a warning if 2+ envvars are setting the key with different values ? Maybe for another PR.

Reply:
Hmm, interesting questions. I think this is fine for now, but I'll investigate for a possible follow-up PR.